### PR TITLE
add new tech compony in Nepalgunj, Named Ninja Infosys

### PR DESCRIPTION
**Pull request by iam-neo**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

### CODE REVIEW

**Comment by iam-neo** [comment]
Added tech compony "ninja infosys"

**Review by iam-neo — COMMENTED** [review]
Added pvt ltd in compony name and company's date of establish is in Unix Timestamp

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Leave a whitespace at the end of the file

**Comment by iam-neo** [comment]
Done please check at once.

File: data/ninja-infosys.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Ninja Infosys Pvt. Ltd.",
+  "establishedAt": 1572459300,
+  "location": "Nepalgunj, Banke",
+  "website": "https://ninjainfosys.com/",
+  "email": "ninjainfosys@gmail.com",
+  "phone": "+977-9858042433",
```
Comment (thenishantsapkota) [inline data/ninja-infosys.json:7]: Does the company have a landline number?
Landline numbers would be much preferred, if not then also it's alright

Comment (iam-neo) [reply data/ninja-infosys.json]: I just updated landline number, please check.

**Review by iam-neo — COMMENTED** [review]
Updated Landline number

File: data/ninja-infosys.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Ninja Infosys Pvt. Ltd.",
+  "establishedAt": 1572459300,
+  "location": "Nepalgunj, Banke",
+  "website": "https://ninjainfosys.com/",
+  "email": "ninjainfosys@gmail.com",
+  "phone": "081-530361",
```
Comment (thenishantsapkota) [inline data/ninja-infosys.json:7]:
```suggestion
  "phone": "+977-81530361"
```

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Review by thenishantsapkota — APPROVED** [review]

**Comment by thenishantsapkota** [comment]
Thank you for your contribution @iam-neo 🙏

**Comment by iam-neo** [comment]
Thank you for your approval,
I'll add more companies in future soon.

**Pull request merged**